certbot-apache/certbot_apache/_internal/interfaces.py

```diff
@@ -131,9 +131,10 @@ class ParserNode(object):
 
     ParserNode objects should have the following attributes:
 
-    # Reference to ancestor node, or None if the node is the root node of the
-    # configuration tree.
-    ancestor: Optional[ParserNode]
+    # Reference to ancestor node(s), empty if the node is the root node of the
+    # configuration tree. This node can have multiple ancestors if it is in
+    # a file included by multiple files.
```

Comment:
Let's add a note here about how callers can't have any expectation around ordering of the ancestors within the tuple.